app/models/content_item.rb

```diff
@@ -219,12 +219,10 @@ def route_set
 private
 
   def should_register_routes?(previous_item: nil)
-    return false if schema_name.start_with?("placeholder")
-
     if previous_item
```

Comment:
Could possibly inline this, but that may be personal preference

Reply:
Nice idea.  Have made that change.